core/templates/pages/blog-dashboard-page/blog-dashboard-page.mainpage.html

```diff
@@ -1,82 +0,0 @@
-<!DOCTYPE html>
-<html ng-app="oppia" itemscope itemtype="http://schema.org/Organization">
-  <head>
-    @load('base-components/header.template.html', {"title": "Blog Dashboard - Oppia"})
-    <script>
-    // This prevents angular pages to cause side effects to hybrid pages.
-    // TODO(#13080): Remove fake resumeBootstrap once angularjs is removed
-    // from this page (schema based editor is migrated to angular).
-    if (!window.angular.resumeBootstrap) {
-      window.angular.resumeBootstrap = function(extraModules) {};
-    }
-    </script>
-    <style>
-      button:focus {
-        border: unset;
-        outline: unset;
-      }
-
-      @media(max-width: 768px) {
-        .collapse:not(.show) {
-          display: flex;
-        }
-
-        .navbar-brand {
-          display: none;
-        }
-
-        .oppia-navbar-brand-name.d-none {
-          /* !important is necessary otherwise d-none class of base components is not overwritten(!important
-          is being used there). */
-          display: flex !important;
-          margin-left: -10px;
-        }
-
-        .ng-cloak {
-          display: flex !important;
-          flex-direction: row;
-        }
-
-        .float-left {
-          margin-left: 5%;
-        }
-      }
-      @media (max-width: 500px) {
-        .float-left {
-          margin-left: 2%;
```

Comment:
Do we need to move these style somewhere?

Reply:
Can we move it to blog-dashboard-root.component.html? (Please test the changes i.e, check how these style affects the page on `develop` and then make sure you see similar effects on your feature branch)

Reply:
Moved it to blog-dashboard-root.component.html, didnt see any changes